dimelo/load_processed.py

```diff
@@ -555,12 +646,29 @@ def readwise_binary_modification_arrays(
                 mod_pos_in_read = np.flatnonzero(read_data[mod_vector_index] > thresh)
 
             if relative:
-                mod_pos_record = (
-                    mod_pos_in_read
-                    + read_data[read_start_index]
-                    - (read_data[region_start_index] + read_data[region_end_index]) // 2
-                )
+                if regions_5to3prime and read_data[region_strand_index] == "-":
+                    # Here we want to show the regions each along their 5 prime to 3 prime direction
+                    # This means that negative strand regions need to be flipped
+                    mod_pos_record = -(
```

Comment:
Is the only difference here multiplying by -1? If so, this whole statement can be simplified to reduce code duplication.

Reply:
Yeah I thought there would be more differences when writing it for some reason, but turned out that was all I needed actually.